Scarb.toml

```diff
@@ -1,15 +1,18 @@
 [package]
 name = "konoha"
 description = "A flexible monolithic governance contract, developed for use with Carmine Options AMM"
-version = "0.3.0"
+version = "0.4.0"
 cairo-version = "2.6.3"
 
 [dependencies]
 cubit = { git = "https://github.com/influenceth/cubit.git", commit = "62756082bf2555d7ab25c69d9c7bc30574ff1ce8" }
```

Comment:
Is cubit dependency needed for the governance?

Reply:
No longer, removed, thanks